### PR TITLE
Gcw 1797 add client side validation for location

### PR DESCRIPTION
**Pull request by viveksriram1**

### CODE REVIEW

File: app/locales/en/translations.coffee [hunk 1]
```diff
@@ -259,6 +259,7 @@ I18nTranslationsEn =
       "invalid_inventory": "Inventory number is invalid."
       "invalid_quantity": "Quantity can not be blank."
       "invalid_description": "Description can not be blank."
+      "invalid_location": "Location can not be blank"
```
Comment (Darshan-Chauhan) [inline app/locales/en/translations.coffee]: @viveksriram1 , add full-stop at the end of the sentence.

File: app/locales/zh-tw/translations.coffee [hunk 2]
```diff
@@ -260,6 +260,7 @@ I18nTranslationsZhTw =
       "invalid_inventory": "存貨號碼不正確"
       "invalid_quantity": "空格不能留空"
       "invalid_description": "空格不能留空"
+      "invalid_location": "Location can not be blank"
```
Comment (Darshan-Chauhan) [inline app/locales/zh-tw/translations.coffee]: ^

**Comment by Darshan-Chauhan** [comment]
@viveksriram1 , added some comment. Just need one minor change.
Otherwise PR is good.
Good work on test cases also. 👍

File: app/locales/en/translations.coffee (hunk 1 above)
Comment (viveksriram1) [reply app/locales/en/translations.coffee]: Sure @Darshan-Chauhan

File: tests/acceptance/receive_package_test.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+module('Receive package', {
+  beforeEach: function() {
+    App = startApp({}, 2);
+    TestHelper.setup();
+    item1 = FactoryGuy.make("item", {state: "accepted"});
+    offer1 = FactoryGuy.make("offer", { state: "received", items: [item1] });
+    package1 = FactoryGuy.make("package", { offerId: parseInt(offer1.id), state: "received", item: item1});
+  },
+  afterEach: function() {
+    Em.run(function() { TestHelper.teardown(); });
```
Comment (namrataukirde) [inline tests/acceptance/receive_package_test.js]: I think here we need Ember instead of Em.

Comment (viveksriram1) [reply tests/acceptance/receive_package_test.js]: Oops.. yes I will change it, thanks.

Comment (viveksriram1) [reply tests/acceptance/receive_package_test.js]: @namrataukirde I checked it , It should be Em.run only. :)

**Pull request merged**